### PR TITLE
Additional steps for local testing with minikube added.

### PR DESCRIPTION
**Pull request by nrekretep**
#### Summary
Additional steps for local testing with minikube added to the README.

#### Ticket Link
No Ticket

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -103,3 +103,39 @@ You don't need to push the mattermost-operator image to DockerHub or any other r
...
+```
+* Now is the right time to build your new mattermost image with
+`make build-image`
+* After the build has finishes the new image is available on the minikube node
+```bash
+$ docker images                  
+REPOSITORY                                TAG                 IMAGE ID            CREATED             SIZE
+mattermost/mattermost-operator            test                e7f1e78a130b        2 minutes ago       49.6MB
+...
+```
```
Comment (cpanato) [inline README.md]: thanks for adding this doc @nrekretep 
but after the user make that what are the next steps? can you describe the flow?

File: README.md [hunk 2]
```diff
@@ -103,3 +103,39 @@ You don't need to push the mattermost-operator image to DockerHub or any other r
 ```bash
 $ kind load docker-image mattermost/mattermost-operator:test
```
Comment (gabrieljackson) [inline README.md]: Doesn't this need to be done after the new instructions? If not, does this line need to be removed?

**Comment by jasonblais** [comment]
Hey @nrekretep, huge thanks for your contribution! Let us know if you have questions about the feedback above. We'd be happy to clarify :)

**Comment by cpanato** [comment]
closing due inactivity

**Comment by cpanato** [comment]
thanks for your contribution, feel free to reopen or submit a new one if you want

**Pull request closed without merging**